app/tests/conftest.py

```diff
@@ -27,12 +32,18 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import Session, sessionmaker
 
+# isort: split - DO NOT MOVE THIS IMPORT, this should be before any app.*
+from .sdk_container_mock import mock_instance  # Mock is already setup when this imports
```

Comment:
Where is this being used?

Reply:
The initial goal was to initialize `mock_instance` before any app.*. 
I moved this import to the `__init__.py`, and it worked same way. Thanks, good catch!